Framework/DataHandling/src/LoadILLSANS.cpp

```diff
@@ -265,6 +298,76 @@ void LoadILLSANS::initWorkSpace(NeXus::NXEntry &firstEntry,
   }
 }
 
+/**
+ * @brief LoadILLSANS::initWorkSpaceD11B Load D11B data
+ * @param firstEntry
+ * @param instrumentPath
+ */
+void LoadILLSANS::initWorkSpaceD11B(NeXus::NXEntry &firstEntry,
+                                    const std::string &instrumentPath) {
+  g_log.debug("Fetching data...");
+
+  NXData data1 = firstEntry.openNXData("data1");
+  NXInt dataCenter = data1.openIntData();
+  dataCenter.load();
+  NXData data2 = firstEntry.openNXData("data2");
+  NXInt dataRight = data2.openIntData();
+  dataRight.load();
+  NXData data3 = firstEntry.openNXData("data3");
+  NXInt dataLeft = data3.openIntData();
+  dataLeft.load();
+
+  size_t numberOfHistograms =
+      static_cast<size_t>(dataCenter.dim0() * dataCenter.dim1() +
+                          dataRight.dim0() * dataRight.dim1() +
+                          dataLeft.dim0() * dataLeft.dim1()) +
+      N_MONITORS;
+
+  createEmptyWorkspace(numberOfHistograms, 1);
+  loadMetaData(firstEntry, instrumentPath);
+
+  size_t nextIndex;
+  nextIndex =
+      loadDataIntoWorkspaceFromVerticalTubes(dataCenter, m_defaultBinning, 0);
```

Comment:
This method dates to prehistoric times. Now the tube directions are defined in IDFs. Maybe we should take the opportunity to rename this to `loadDataFromTubes`?

Reply:
Done, it's clearer indeed